frappe/desk/page/setup_wizard/setup_wizard.py

```diff
@@ -248,6 +249,12 @@ def update_user_name(args):
 		add_all_roles_to(args.get("name"))
 
 
+def set_timezone(args):
+	if args.get("timezone"):
+		for name in frappe.STANDARD_USERS:
+			frappe.db.set_value("User", name, "time_zone", args.get("timezone"))
```

Comment:
This problem will apply to first created user too?

Reply:
I thought about it, yes. But then I figured, a non-Administrator first user is only possible with custom code, right? So the custom code may do whatever fits their needs?

frappe.STANDARD_USERS is a wonderful solution though. Let’s do it this way! 👍

Reply:
Yeah I checked, the first user gets created after system settings are updated, so it's unaffected.